dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1290,7 +1290,7 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		CharacterComponent* character = entity->GetComponent<CharacterComponent>();
 		if (character) character->SetUScore(character->GetUScore() + uscore);
 
-		GameMessages::SendModifyLEGOScore(entity, entity->GetSystemAddress(), uscore, LOOTTYPE_NONE);
+		GameMessages::SendModifyLEGOScore(entity, entity->GetSystemAddress(), uscore, LOOT_SOURCE_MODERATION);
```

Comment:
Why `LOOT_SOURCE_MODERATION`?

Reply:
This line is sent from the SlashCommandHandler and I deemed this as UScore being earned from a moderation standpoint hence the loot source.

Reply:
I made a similar change in this file when fixing #291 for coins being earned through /setcurrency.  If this should also be changed let me know what they should be changed to.

Reply:
I've noticed through testing that the client does not update their uscore when this is set to moderation or `LOOT_SOURCE_NONE`.  What's strange is the achievements and missions work just fine when being sent their respective loot sources.

Reply:
Changed the value here to LOOT_SOURCE_CHEST (1) since other loot sources did not work as per a discussion with lcdr